src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,343 @@
+	"TracerMolecularWeightUnit": "g/mol",
+	"TracerInjectionType": "bolus",
+	"InjectedRadioactivity": 573,
+	"InjectedRadioActivityUnit": "MBq",
+	"InjectedMass": 0.62,
+	"InjectedMassUnit": "ug",
+	"SpecificRadioactivity": 929.6,
+	"SpecificRadioactivityUnit": "MBq/ug",
+	"ModeOfAdministration": "bolus",
+	"MolarActivity": 353.51,
+	"MolarActivityUnit": "GBq/umol",
+	"MolarActivityMeasTime": "13:04:42",
+	"TimeZero": "13:04:42",
+	"ScanStart": 0,
+	"InjectionStart": 0,
+	"FrameTimesStart": [0, 10, 20, 30, 40, 50, 60, 80, 100, 120, 140, 160, 180, 240, 300, 360, 420, 480, 540, 660, 780, 900, 1020, 1140, 1260, 1380, 1500, 1800, 2100, 2400, 2700, 3000, 3300, 3600, 3900, 4200, 4500, 4800, 5100, 5400, 5700, 6000, 6300, 6600, 6900],
+	"FrameDuration": [10, 10, 10, 10, 10, 10, 20, 20, 20, 20, 20, 20, 60, 60, 60, 60, 60, 60, 120, 120, 120, 120, 120, 120, 120, 120, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300],
+	"AcquisitionMode": "list mode",
+	"ImageDecayCorrected": true,
+	"ImageDecayCorrectionTime": 0,
+	"ReconMethodName": "3D-OSEM-PSF",
+	"ReconMethodParameterLabels": ["subsets","iterations"],
+	"ReconMethodParameterUnit": ["none","none"],
+	"ReconMethodParameterValues": [16,10],
+	"ReconFilterType": "none",
+	"AttenuationCorrection": "[137Cs]transmission scan-based"
+}
+```
+
+## Blood recording data
+This section includes all data needed to perform blood analysis for PET data. The section may be omitted completely if blood measurements of radioactivity were not made. If blood measurements were made a *_blood.json file is expected that contains information about which blood data is available as well as some blood measurement details. Furthermore, depending on the information given in the *_blood.json file we expect .tsv files with corresponding .json files that contain the values and information regarding plasma samples, metabolite fractions, discrete blood samples and/or continuous blood samples from an autosampler. All these measurements should be defined according to a single time-scale in relation to time zero defined by the PET data (Figure 1). Additionally, if blood and metabolite measurements were made one should always report radioactivity in plasma and corresponding parent fraction measurements, including fractions of radiometabolites. All definitions used below are in accordance with Innis et al. 2007 [3]. All method specific information related to the measurements can be stated in the field “description”.
+
+Template:
+
+```Text
+sub-<participant_label>/
+ 	[ses-<session_label>/]
+	pet/
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_blood.json
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_recording-<label>.tsv
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_recording-<label>.json
```

Comment:
These do not appear to have a suffix.

Reply:
Hi @tsalo, so we were not sure if the .tsv files and accompanying .json files should have a suffix. This we had discussed a bit with @robertoostenveld. We want to be able to check the .tsv and .json files in the validator and otherwise our naming would conflict with the _blood.json file which ahs all the required meta information about blood.

Reply:
Hi @tsalo, did my answer above make sense?

Reply:
I'm so sorry for not responding to this! From what I understand, a suffix is an absolutely essential part of BIDS filenames, so I think we need to find a solution where all files have suffixes. That said, I need to re-read the BEP in a bit more detail before I can come up with any ideas.

Reply:
This has been resolved with the maintainers.